eng/tools/summarize-impact/src/cli.ts

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env node
+
+import { evaluateImpact } from "./impact.js";
+import { getChangedFilesStatuses } from "@azure-tools/specs-shared/changed-files";
+import { setOutput } from "@azure-tools/specs-shared/error-reporting";
+
+import { resolve, join } from "path";
+import fs from "fs";
+import { parseArgs, ParseArgsConfig } from "node:util";
+import { simpleGit } from "simple-git";
+import { LabelContext, PRContext } from "./types.js";
+
+export async function getRootFolder(inputPath: string): Promise<string> {
+  try {
+    const gitRoot = await simpleGit(inputPath).revparse("--show-toplevel");
+    return resolve(gitRoot.trim());
+  } catch (error) {
+    console.error(
+      `Error: Unable to determine the root folder of the git repository.`,
+      `Please ensure you are running this command within a git repository OR providing a targeted directory that is within a git repo.`,
+    );
+    process.exit(1);
+  }
+}
```

Comment:
This should probably move to .github/shared.  Maybe create file `.github/shared/simple-git.js` we can use for this and similar helpers?